### PR TITLE
I18n works

### PR DESCRIPTION
**Pull request by longhotsummer**
The major idea here is that all the field labels and titles are driven by the form, rather than repeating it everywhere.

### CODE REVIEW

**Comment by actlikewill** [comment]
@longhotsummer I think I see a few missed errors like this one https://github.com/laws-africa/indigo/blob/f08ff7920a26f1c99d38f0b1459a01b919000f17/indigo_app/forms/works.py#L551

File: indigo_app/forms/works.py [hunk 1]
```diff
@@ -1025,6 +1043,17 @@ def filter_queryset(self, queryset, exclude=None):
 
         return queryset.distinct()
 
+    def facet_item(self, field, value, count):
+        try:
+            label = next(lab for (val, lab) in self.fields[field].choices if val == value)
+        except StopIteration:
+            raise ValueError(f"Unknown choice {value} for field {field}")
```
Comment (actlikewill) [inline indigo_app/forms/works.py:1050]: does this error need translation?

**Comment by actlikewill** [comment]
these two:
https://github.com/laws-africa/indigo/blob/f08ff7920a26f1c99d38f0b1459a01b919000f17/indigo_app/forms/works.py#L551
https://github.com/laws-africa/indigo/blob/f08ff7920a26f1c99d38f0b1459a01b919000f17/indigo_app/forms/works.py#L470

File: indigo_app/forms/works.py (hunk 1 above)
Comment (longhotsummer) [reply indigo_app/forms/works.py:1050]: no, it's not user facing, it will raise a 500-level error. It's to guard against typos lower down.

**Comment by longhotsummer** [comment]
@actlikewill updated, thanks!

**Pull request merged**